examples/http-1-1-to-2-0-protocol-switch/http_1_1_to_2_0_protocol_switch.bal

```diff
@@ -19,10 +18,9 @@ service / on new http:Listener(9090) {
 }
 
 // HTTP version is set to 2.0.
-listener http:Listener http2serviceEP = new (7090,
-    config = {httpVersion: "2.0"});
+listener http:Listener http2serviceEP = new (7090);
```

Comment:
What is the usage of `http2serviceEP`?

Reply:
Fixed with [18d6eb246533e444b42727ad2010b376160c5cb1](https://github.com/ballerina-platform/ballerina-distribution/pull/3024/commits/18d6eb246533e444b42727ad2010b376160c5cb1)